velox/docs/monthly-updates/december-2021.rst

```diff
@@ -0,0 +1,40 @@
+********************
+December 2021 Update
+********************
+
+Documentation
+-------------
+
+* Add :doc:`Aggregation-related optimizations <../develop/aggregations>` guide.
+* Add :doc:`json_extract_scalar <../functions/json>` function.
+* Update :doc:`Mathematical Functions <../functions/math>`.
```

Comment:
perhaps, clarify as "Adding documentation for missing functions in :doc:`Mathematical Functions <../functions/math>` and :doc:`Comparison Functions <../functions/comparison>`"